lib/install.js

```diff
@@ -144,6 +147,15 @@ exports.install = function(targets, options) {
     .then(function() {
       return saveInstall();
     })
+    .then(function () {
+      var hooks = config.pjson.hooks;
+      if (hooks) {
+        if ('postinstall' in hooks && typeof hooks.postinstall === 'string') {
+          System.config(config.getLoaderConfig());
+          return api.import(hooks.postinstall);
```

Comment:
If we use this pattern we will cache the loader configuration which may cause issues with ensuring we use the latest configurations.

It may be better to actually use `var Loader = api.Loader` and then `return new Loader().import(hooks.postinstall)`.

Also we should include a catch handler that gives some error information about the hook failure.